### PR TITLE
Changelog and Readme Update

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -745,7 +769,7 @@ Support to provide custom or organizational CA signed certificate for SSL valida
 Major Changes
 -------------
 
-- All modules can read custom or organizational CA signed certificate from the environment variables. Please refer to `SSL Certificate Validation <https://github.com/dell/dellemc-openmanage-ansible-modules#ssl-certificate-validation>` _ section in the `README.md <https://github.com/dell/dellemc-openmanage-ansible-modules /blob/collections/README.md#SSL-Certificate-Validation>` _ for modification to existing playbooks or setting environment variable.
+- All modules can read custom or organizational CA signed certificate from the environment variables. Please refer to `SSL Certificate Validation <https://github.com/dell/dellemc-openmanage-ansible-modules#ssl-certificate- validation>` _ section in the `README.md <https://github.com/dell/dellemc-openmanage-ansible-modules /blob/collections/README.md#SSL-Certificate-Validation>` _ for modification to existing playbooks or setting environment variable.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- All modules can read custom or organizational CA signed certificate from the environment variables. Please refer to `SSL Certificate Validation <https://github.com/dell/dellemc-openmanage-ansible-modules#ssl-certificate-validation>` _ section in the `README.md <https://github.com/dell/dellemc-openmanage-ansible-modules /blob/collections/README.md#SSL-Certificate-Validation>` _ for modification to existing playbooks or setting environment variable.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,30 @@ Dell OpenManage Ansible Modules Release Notes
...
 
 
+v8.5.0
+======
+
+Release Summary
+---------------
+
+- Ansible lint issues fix for roles.
+- Enhancement to add job tracking till completion and enable apply time on the controller that is supported for redfish_storage_volume.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]: redfish_storage_volume module is enhanced to support reboot option, and job tracking

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,30 @@ Dell OpenManage Ansible Modules Release Notes
 .. contents:: Topics
 
 
+v8.5.0
+======
+
+Release Summary
+---------------
+
+- Ansible lint issues fix for roles.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]:
```suggestion
- Ansible lint issue fix for roles.
```

File: CHANGELOG.rst (hunk 1 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: Done.

File: CHANGELOG.rst (hunk 3 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: Done.

File: galaxy.yml [hunk 4]
```diff
@@ -55,3 +55,4 @@ documentation: https://github.com/dell/dellemc-openmanage-ansible-modules/tree/c
 build_ignore:
   - BuildNumber.txt
   - codecov.yml
+  - playbook/demo
```
Comment (sachin-apa) [inline galaxy.yml]:
```suggestion
```

**Review by sachin-apa — DISMISSED** [review]

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,30 @@ Dell OpenManage Ansible Modules Release Notes
 .. contents:: Topics
 
 
+v8.5.0
+======
+
+Release Summary
+---------------
+
+- Ansible lint issues are fixed for Redfish roles.
```
Comment (sachin-apa) [inline CHANGELOG.rst]: Ansible lint issues are fixed for the collections

File: CHANGELOG.rst [hunk 6]
```diff
@@ -61,9 +85,11 @@ Known Issues
 
 - ca_path missing - Issue(275740) - The roles idrac_attributes, redfish_storage_volume, and idrac_server_powerstate have a missing parameter ca_path.
 - idrac_firmware - Issue(276335) - This module fails on the Python 3.11.x version with NFS shares. Use a different Python version or Share type.
+- idrac_firmware - Issue(276335) - This module fails on the Python 3.11.x version with NFS shares. Use a different Python version or Share type.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]: Remove this

Comment (sachin-apa) [inline CHANGELOG.rst]: why there are two entries here?

File: CHANGELOG.rst [hunk 7]
```diff
@@ -61,9 +85,11 @@ Known Issues
...
 - ome_device_network_services - Issue(212681) - The module does not provide a proper error message if unsupported values are provided for the following parameters- port_number, community_name, max_sessions, max_auth_retries, and idle_timeout.
 - ome_device_power_settings - Issue(212679) - The module displays the following message if the value provided for the parameter ``power_cap`` is not within the supported range of 0 to 32767, ``Unable to complete the request because PowerCap does not exist or is not applicable for the resource URI.``
 - ome_device_quick_deploy - Issue(275231) - This module does not deploy a new configuration to a slot that has disabled IPv6.
+- ome_device_quick_deploy - Issue(275231) - This module does not deploy a new configuration to a slot that has disabled IPv6.
```
Comment (rajshekarp87) [inline CHANGELOG.rst]: Remove this

Comment (sachin-apa) [inline CHANGELOG.rst]: why there are two entries here

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: CHANGELOG.rst (hunk 5 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: done

File: CHANGELOG.rst (hunk 6 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: removed

File: CHANGELOG.rst (hunk 7 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: removed

**Review by sachin-apa — DISMISSED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

File: CHANGELOG.rst (hunk 6 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: done

File: CHANGELOG.rst (hunk 7 above)
Comment (ShivamSh3) [reply CHANGELOG.rst]: done

**Review by rajshekarp87 — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Pull request merged**